rgd/geodata/models/imagery/etl.py

```diff
@@ -35,6 +45,9 @@
 
 logger = get_task_logger(__name__)
 
+GDAL_DATA = os.path.join(os.path.dirname(rasterio.__file__), 'gdal_data')
+os.environ['GDAL_DATA'] = GDAL_DATA
```

Comment:
Might need to also set a PROJ env variable for NTIF format issue in _reproject_raster

Reply:
Will look into this later.